doc/man1/openssl-cmp.pod.in

```diff
@@ -629,7 +630,8 @@ The client's current CMP signer certificate.
 Requires the corresponding key to be given with B<-key>.
 The subject of this certificate will be used as sender of outgoing CMP messages,
 while the subject of B<-oldcert> or B<-subjectName> may provide fallback values.
-The issuer of this certificate is used as one of the recipient fallback values.
+The issuer of this certificate is used as one of the recipient fallback values
+and as fallback issuer entry in the certificate template of IR, CR, and KUR.
```

Comment:
Not sure why you use 2 different formats
IR, CR and KUR.
but on line 407 you do IR/CR/KUR?

Reply:
Good point to better be consistent here: I've changed this (and also in **`OSSL_CMP_CTX_new.pod`) to `IR/CR/KUR` used elsewhere, which is also more concise.